src/model/invite.rs

```diff
@@ -46,6 +46,24 @@ pub struct Invite {
     /// This can be [`None`] for invites created by Discord such as invite-widgets
     /// or vanity invite links.
     pub inviter: Option<InviteUser>,
+
+    /// The type of target for this voice channel invite.
+    pub target_type: Option<InviteTargetType>,
+    /// The user whose stream to display for this voice channel stream invite.
+    ///
+    /// Only shows up if `target_type` is `Stream`.
+    pub target_user: Option<UserId>,
+    /// The embedded application to open for this voice channel embedded application invite.
+    ///
+    /// Only shows up if `target_type` is `EmmbeddedApplication`.
+    pub target_application: Option<ApplicationId>,
+
+    // /// The expiration date of this invite, returned from `Http::get_invite` when
+    // /// `with_expiration` is true.
+    // pub expires_at: Option<DateTime<Utc>>,
```

Comment:
I presume the extra slashes are accidental?
```suggestion
    /// The expiration date of this invite, returned from `Http::get_invite` when
    /// `with_expiration` is true.
    pub expires_at: Option<DateTime<Utc>>,
```

Reply:
No, the Http method doesn't support providing that field, so I commented it out.
I wanted this to be non-breaking, and if expires_at would always be None, there's no point in implementing it. The code is still there for some possible breaking change though (different PR after this?)